src/legacy/core_plugins/region_map/public/region_map_vis_params.js

```diff
@@ -17,80 +17,79 @@
  * under the License.
  */
 
-import { uiModules } from 'ui/modules';
 import { toastNotifications } from 'ui/notify';
 import regionMapVisParamsTemplate from './region_map_vis_params.html';
 import { mapToLayerWithId } from './util';
-import '../../tile_map/public/editors/wms_options';
+
+// TODO: reference to TILE_MAP plugin should be removed
 import { ORIGIN } from '../../../../legacy/core_plugins/tile_map/common/origin';
```

Comment:
@lukeelmers  @nreese 
In this PR I've marked all places where we do an import from tile_map plugin. Not sure that we need to fix it now because in future we are going to create one common plugin for tile_map and region_map

Reply:
I agree this is probably outside the scope of this PR.

We could open another PR that exports it (and any other stuff imported from it) from `tile_map/public/index` if you think it will make it easier to remember later.
 
But I don't think it's critical to address right now unless @nreese feels otherwise.